steemconnect4j/src/test/java/com/hapramp/steemconnect4j/tests/ExampleTest.java

```diff
@@ -0,0 +1,19 @@
+package com.hapramp.steemconnect4j.tests;
+
+import com.hapramp.steemconnect4j.SteemConnect;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class ExampleTest {
+    @Test
+    public void testMe() {
```

Comment:
This is not a good name I guess. Also, we do not need this test as the actual feature is not implemented.